atd-vze/src/views/Tools/CreateCrashRecordTable.js

```diff
@@ -85,11 +75,11 @@ const CreateCrashRecordTable = ({ crashesData, loading, error }) => {
   };
 
   /**
-   * Commits the crash_id to be deleted to state, and prompts for deletion.
-   * @param {int} crashId - The crash id to be deleted.
+   * Commits the crash record id to be deleted to state, and prompts for deletion.
+   * @param {int} recordId - The record id to be deleted.
    */
-  const openModalDelete = crashId => {
-    setDeleteId(crashId);
+  const openModalDelete = recordId => {
+    setDeleteId(recordId);
     toggleModalDelete();
```

Comment:
Would you mind updating the warning message inside the modal? Remove "This cannot be undone" since that is no longer true.

<img width="516" alt="warning" src="https://github.com/user-attachments/assets/92c9688b-2858-42f1-99e4-13221f70e862">